net-print/gtklp/gtklp-1.3.1.ebuild

```diff
@@ -15,14 +15,13 @@ SLOT="0"
 KEYWORDS="amd64 ppc ~sparc x86"
 IUSE="nls ssl"
 
-RDEPEND="x11-libs/gtk+:2
+RDEPEND="
 	net-print/cups
+	x11-libs/gtk+:2
 	nls? ( sys-devel/gettext )
 	ssl? ( dev-libs/openssl:0= )"
-DEPEND="${RDEPEND}
-	virtual/pkgconfig"
-
-DOCS="AUTHORS BUGS ChangeLog README TODO USAGE"
```

Comment:
are you losing `USAGE` with this? consider putting `dodoc USAGE` after the default in `src_install` if thats the case

Reply:
Done